asciidoc/__init__.py

```diff
@@ -2,5 +2,6 @@
 
 from .api import AsciiDocAPI
 from .asciidoc import execute, cli
+from .asciidoc import VERSION
 
-__all__ = ['AsciiDocAPI', 'execute', 'cli']
+__all__ = ['AsciiDocAPI', 'execute', 'cli', VERSION]
```

Comment:
just for point of reference, why do you need the version here exposed at `__init__.py` as opposed to being under `asciidoc.asciidoc.VERSION`?

Reply:
It's easier to import if you need to get the version's value.  Also, it ought to apply to the entire package instead of just the asciidoc.py file.  I thought of moving the metadata import out of asciidoc.py into __init__.py, but I thought that was more refactoring than we need for this PR.

The only reason I want to get the version right now is to patch the current pip-imported package and my plan is to do a conditional monkey-patch.  When the installed version catches up, then the patch won't be applied.

In the future there might be other reasons to want to get the version number, who knows?